github/misc.go

```diff
@@ -158,6 +158,10 @@ type APIMeta struct {
 	// An array of IP addresses in CIDR format specifying the addresses
 	// which serve GitHub Pages websites.
 	Pages []string `json:"pages,omitempty"`
+
+	// An Array of IP addresses specifying the addresses that source imports
+	// will originate from on GitHub.com
```

Comment:
Minor, but there should be a period at the end of this sentence. We want to have proper grammar and consistency.

Reply:
Fixed!